lua/plenary/job.lua

```diff
@@ -84,6 +84,10 @@ function Job:new(o)
   obj.args = o.args
   obj.cwd = o.cwd and vim.fn.expand(o.cwd)
   obj.env = o.env
+  if o.interactive == nil then
+    obj.interactive = true
+  else
+    obj.interactive = o.interactive
```

Comment:
I think you need a new `end` here.

Reply:
Uh yea, I worked on the main branch to test it since thats where I had telescope etc, must've missed this when I copied it over..